src/sentry/utils/committers.py

```diff
@@ -259,9 +258,4 @@ def get_serialized_event_file_committers(project, event, frame_limit=25):
 
 
 def dedupe_commits(commits):
-    result = {}
-    for obj in commits:
-        if obj["id"] not in result:
-            result[obj["id"]] = obj
-
-    return result.values()
+    return {c["id"]: c for c in commits}.values()
```

Comment:
The semantics here are slightly different. Before we used the _first_ id, how duplicate id's will be the _last_ id.

Reply:
I don't understand what you're saying?

Reply:
Okay, this was briefly discussed in Slack, I understand it now. Surely we don't care about ordering here since the goal is deduping.

Reply:
Well it matters what the commit objects look like, if all the values in an object with the same `id` are guaranteed to be the same, then it's good, but if not then it could be problematic.

Reply:
@scefali any thoughts here?

Reply:
@EvanPurkhiser This change is fine, it doesn't matter which one we use in de-duping. The objects are the same.

Reply:
👍